src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4088,6 +4088,7 @@ internal static string GetClearHostFunctionText()
                 // Porting note: non-Windows platforms use `clear`
                 return @"
 & (Get-Command -CommandType Application clear | Select-Object -First 1).Definition
+""`e[3J"" # clear the scrollback buffer
```

Comment:
Better to use `$host.ui.Write("`e[3J")`, because direct output `"`e[3J"` invariably adds a trailing newline, which means that the prompt on the cleared screen will appear on the _2nd_ line.